Week2/todo_app/program.js

```diff
@@ -2,7 +2,7 @@
 // that we can use to connect to a MySQL process
 // and access its databases.
 const mysql = require('mysql');
-
+const config = require('./db-secret.json');
```

Comment:
Good use of the config file technique (did you use gitignore to keep it out of the repo?).

Reply:
Yeah I used gitignore to store my password.